### DISCUSSION TITLE
Could I update the pulsar-client version?

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#30080] [Q&A and General discussion]
There is an old version (github.com/apache/pulsar-client-go v0.6.1-0.20210728062540-29414db801a7)
I want to update it to v0.11.0 to enable the Consumer Option's param 'ReplicaSubscriptionState'

Whether that's possible or not

**Comment by yhmo** [comment]
Not sure.
The version v0.6.1 is tested a lot and stable for the current milvus. If you change the pulsar client to 0.11.0, you need to compile and test it by yourself.

**Reply by bingbb47** [reply]
Okay, this might be more complicated than I thought. 🥹
